xarray_sentinel/esa_safe.py

```diff
@@ -21,6 +22,39 @@
 }
 
 
+def get_annotation_path(
+    product_path: T.Union[str, "os.PathLike[str]"],
+    subswath: str,
+    polarization: str = "vv",
+) -> "os.PathLike[str]":
+    manifest = open_manifest(product_path)
+    product_attrs, product_files = parse_manifest_sentinel1(manifest)
+
+    product_path = pathlib.Path(product_path)
```

Comment:
I would prefer open_manifest to return the `manifest_path` (as a `pathlib.Path`) as well so you can just do `folder = manifest_path.parent`.

E.g.: `manifest_path, manifest = open_manifest(product_path)`